### DISCUSSION TITLE
Search Order (Set search title before document contents)

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2901] [Q&A]
Is there any way to set which property is searched first?
For example when searching "Medical" or "Sliding Fee" in the search bar, this is the results.
![image](https://user-images.githubusercontent.com/22782498/231293494-8bc654c3-ffce-4c19-8939-10becf1723d1.png)
![image](https://user-images.githubusercontent.com/22782498/231294196-77ac479d-d99c-4092-b909-502e218c4869.png)

How can we make it so those documents that have 'medical' or 'Sliding Fee' in the title are listed first before ones that are included because medical is somewhere in the contents?

**Comment by kasperbolarsen** [comment]
To me it sounds like you are looking for a way to rank your results. XRANK might be the solution, try to read this primer https://searchexplained.com/how-to-sort-search-results-by-using-query-rules/

**Reply by nicolesherman00** [reply]
Is this something you can do in SP online ? My XRank option is greyed out.
